slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/OptionBooleanTest.scala

```diff
@@ -0,0 +1,29 @@
+package com.typesafe.slick.testkit.tests
+
+import com.typesafe.slick.testkit.util.{AsyncTest, JdbcTestDB}
+import slick.jdbc.OracleProfile
+
+class OptionBooleanTest extends AsyncTest[JdbcTestDB] {
+  lazy val oracleProfile = tdb.profile.asInstanceOf[OracleProfile]
```

Comment:
Can we change this test so that we can execute this for all databases? This is something that should work regardless of the profile that is used.

Reply:
Yes, I've made appropriate changes.